modules/Lending.py

```diff
@@ -171,6 +173,9 @@ def lend_cur(active_cur, total_lended, lending_balances):
 
     loans = api.return_loan_orders(active_cur, loanOrdersRequestLimit[active_cur])
     loans_length = len(loans['offers'])
+    if hide_coins and Decimal(loans['offers'][0]['rate']) < Decimal(cur_min_daily_rate):
```

Comment:
Will solve this in rewrite, if issues starting coming up soon I may put a try-except around it.